src/charts/stackedArea/StackedAreaComponent.js

```diff
@@ -56,7 +56,8 @@ export default class StackedArea extends React.Component {
     }
 
     componentDidUpdate() {
-        this.props.chart.update(this._rootNode, this.props.data, this._getChartConfiguration());
+        this.props.chart.destroy(this._rootNode);
+        this.props.chart.create(this._rootNode, this.props.data, this._getChartConfiguration());
```

Comment:
`chart.update` doesn't work for passing `width`, so this breaks the responsive container we built.

However, I had a very hard time getting `withResponsiveness` to work correctly together with `TooltipComponent` and `StackedAreaComponent`. It seems that I can only get either the tooltip to work, or the responsiveness, but not both. I'm going to spend more time on it tmrw but we may want to consider an alternate way of doing responsiveness.